source/_components/built-with-jigsaw.php

```diff
@@ -0,0 +1,14 @@
+<section class="bg-gradient-purple py-16">
+    <div class="flex flex-col text-center px-2 sm:px-0">
+        <h2 class="text-white mb-2">Need inspiration?</h2>
+
+        <p class="text-pink-lighter mb-8">Browse websites built with Jigsaw, find articles, or get featured.</p>
+
+        <img src="/assets/img/browsers-illustration.svg" alt="Built with Jigsaw by Tighten" />
+
+        <a href="https://builtwithjigsaw.com" title="Showcase of websites built with Jigsaw"
+            class="mx-auto py-4 px-6 text-grey-darker no-underline tracking-wide uppercase rounded-lg text-sm bg-grey-lightest shadow-lg">
+            Visit Built With Jigsaw
```

Comment:
This reads awkwardly; we should bold `Built with Jigsaw` or use a different color